src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -55,8 +55,9 @@ public void WriteClient(CodeWriter writer, LowLevelClient client)
                     WriteConstructors(writer, client);
 
                     var exampleComposer = new LowLevelExampleComposer(client);
-                    foreach (var clientMethod in client.ClientMethods)
+                    foreach (var convenienceMethod in client.ConvenienceMethods)
```

Comment:
We'll filter convenience methods through configuration. So here we cannot assume a 1:1 mapping.